setup.sh

```diff
@@ -464,7 +464,7 @@ fi
 
 if [ $INSTALL -eq 1 ]
 then
-	if [ -z "$VIRTUAL_ENV" ] && [ $UNATTENDED != 1 ]
+	if [ -z "$VIRTUAL_ENV" ] && [ $UNATTENDED != 1 ] && [ -z "$CONDA_DEFAULT_ENV" ]
```

Comment:
Can the conda check go ahead of the unattended check so that like-checks are grouped?

Reply:
Changed the order